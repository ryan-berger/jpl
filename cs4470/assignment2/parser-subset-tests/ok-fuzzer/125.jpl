show 95
print ""
print "ra"
return 20.
let d1J_ = Y3T.
read image "" to U1S_0I.5G.
show 9
show 0
return 4
