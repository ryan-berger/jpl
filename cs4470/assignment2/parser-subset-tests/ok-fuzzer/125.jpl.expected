(ShowCmd (IntExpr 95))
(PrintCmd "")
(PrintCmd "ra")
(StmtCmd (ReturnStmt (FloatExpr 20)))
(StmtCmd (LetStmt (ArgLValue (VarArgument d1J_)) (VarExpr Y3T.)))
(ReadImageCmd "" (VarArgument U1S_0I.5G.))
(ShowCmd (IntExpr 9))
(ShowCmd (IntExpr 0))
(StmtCmd (ReturnStmt (IntExpr 4)))
